src/umpire/strategy/DynamicPool.cpp

```diff
@@ -33,17 +33,17 @@ DynamicPool::DynamicPool(const std::string& name,
                          const int align_bytes) noexcept :
   AllocationStrategy(name, id),
   m_allocator{allocator.getAllocationStrategy()},
-  m_min_alloc_bytes{min_alloc_bytes},
+  m_min_alloc_bytes{round_up(min_alloc_bytes, align_bytes)},
```

Comment:
Where do we confirm that `align_bytes` is valid?  What if I set `align_bytes=21`, will that work?

Reply:
What does "valid" mean?

Reply:
We make the rules for that since it is our API.  I will write a Jira ticket on this.  I'm no positive, but it it would not surprise me if round_up assumed align_bytes at least needed to be a power of 2.

Reply:
I wrote a Jira ticket.  We can decide when/if/where to address this later.